src/lf_apply.rs

```diff
@@ -341,6 +341,23 @@ pub(crate) unsafe fn rav1d_copy_lpf<BD: BitDepth>(
     }
 }
 
+/// Slice `[u8; 4]`s from `lvl`, but "unaligned",
+/// meaning the `[u8; 4]`s can straddle
+/// adjacent `[u8; 4]`s in the `lvl` slice.
+///
+/// Note that this does not result in actual unaligned reads,
+/// since `[u8; 4]` has an alignment of 1.
+/// This optimizes to a single slice with a bounds check.
+#[inline(always)]
+fn unaligned_lvl_slice(lvl: &[[u8; 4]], y: usize) -> &[[u8; 4]] {
+    // Safety: `[u8; 4]` is transmutable to `u8`s.
+    let (_, lvl, _) = unsafe { lvl.align_to::<u8>() };
+    let lvl = &lvl[y..];
+    // Safety: `[u8; 4]` is transmutable to `u8`s.
```

Comment:
Should that read "`u8` is transmutable to `[u8;4]`s" ?

Reply:
Yeah, kinda, but I need to reword it a bit, since a single `u8` of course isn't transmutable to a while `[u8; 4]`.